src/elements/content-sidebar/Sidebar.js

```diff
@@ -60,46 +67,81 @@ const Sidebar = ({
     fileId,
     getPreview,
     getViewer,
+    hasActivityFeed,
+    hasDetails,
+    hasMetadata,
+    hasSkills,
+    isOpen,
     metadataSidebarProps,
     onVersionHistoryClick,
-    selectedView,
 }: Props) => (
-    <React.Fragment>
-        {selectedView === SIDEBAR_VIEW_DETAILS && (
-            <LoadableDetailsSidebar
-                key={fileId}
-                fileId={fileId}
-                onVersionHistoryClick={onVersionHistoryClick}
-                {...detailsSidebarProps}
-                startMarkName={MARK_NAME_JS_LOADING_DETAILS}
-            />
-        )}
-        {selectedView === SIDEBAR_VIEW_SKILLS && (
-            <LoadableSkillsSidebar
-                key={file.id}
-                file={file}
-                getPreview={getPreview}
-                getViewer={getViewer}
-                startMarkName={MARK_NAME_JS_LOADING_SKILLS}
-            />
-        )}
-        {selectedView === SIDEBAR_VIEW_ACTIVITY && (
-            <LoadableActivitySidebar
-                currentUser={currentUser}
-                file={file}
-                onVersionHistoryClick={onVersionHistoryClick}
-                {...activitySidebarProps}
-                startMarkName={MARK_NAME_JS_LOADING_ACTIVITY}
-            />
-        )}
-        {selectedView === SIDEBAR_VIEW_METADATA && (
-            <LoadableMetadataSidebar
-                fileId={fileId}
-                {...metadataSidebarProps}
-                startMarkName={MARK_NAME_JS_LOADING_METADATA}
-            />
-        )}
-    </React.Fragment>
+    <Switch>
+        <Route exact path="/">
```

Comment:
could this be a SidebarRoute as well? Looks like it will handle the pathFallback redirect for us here.

Reply:
I tried this, but it isn't really suitable. It doesn't need to check `enabled`, nor does it have a `render` prop.